diff --git a/etc/dnf/plugins/copr.conf b/etc/dnf/plugins/copr.conf
index bc29926..a90cf46 100644
--- a/etc/dnf/plugins/copr.conf
+++ b/etc/dnf/plugins/copr.conf
@@ -3,6 +3,10 @@ hostname = copr.fedorainfracloud.org
 protocol = https
 port = 443
 
+[main]
+releasever = 37
+distribution = Fedora
+
 # To add another Copr hub, make a new section with its hostname, protocol and port.
 # (protocol and port are optional and their default values are `https` and `443` respectively).
 # [name-of-hub]
-- 
